### DISCUSSION TITLE
Compatibility issues in version 17.9.3 of vs2022

### DISCUSSION THREAD

**Discussion by Nick-Suo** [PrismLibrary/Prism#3105] [WPF]
I recently found a problem using vs2022.

I used the latest prism in this version and found that the InitializeComponent method cannot be associated with the View. When I rolled vs2022 back to 17.9.2, the method took effect again.

I am not sure whether it is vs2022, so I want to ask others if this problem has occurred.

**Comment by thomasgalliker** [comment]
- Make sure the xaml and the xaml.cs inherit from the same base class
- Make sure the csproj does not contain anything related to this xaml file
- Close VS2022, delete obj+bin folders (I use this script: https://gist.github.com/thomasgalliker/25503165c3058225ae20)

This usually helps. I thinkt its not related to Prism.

**Reply by Nick-Suo** [reply]
Thank you for your reply. I will try to use these operations to solve this problem.
